### ISSUE TITLE
4K Video example !

### ISSUE THREAD

**Issue by ttoinou** [yusuketomoto/chainer-fast-neuralstyle#70]
Hi all !

I created two 4K videos with this repo, you can watch them here : https://www.youtube.com/watch?v=2YRVt80g2Ek   and https://www.youtube.com/watch?v=i69cBYI6f-w (watch in 1080p or 4K)

Thanks everyone here for theirs advices :D !

**Comment by 6o6o** [comment]
This is certainly an impressive result, especially the seamless tiling you managed to achieve. But wouldn't the 4K rendering make the features far too small to be any useful? I prefer to process smaller resolution, like 960x540 and then upscale to full HD with waifu2x.

Btw, here's my latest fan music video I made this way: https://youtu.be/gq6q-xRbNxk
It's only 720p, but you get the point.

I'm currently working on the movie titled "Delete My Photos". If everything goes well and it gets approved be the program directors, it's gonna be exciting to see this tech on the big screen!

**Comment by ttoinou** [comment]
Yes you're right the features are too small, this is not watchable below 1080p :-( (youtube compression is also horrible)
I should try to add waifu2X and optical flow in my workflow..
Oh great you're in the TV industry ?

**Comment by 6o6o** [comment]
No, but I'm involved into app development, called [Lucid](http://lucid.pictures). Got an offer to join their team back in the summer. Didn't refuse, since it was a great opportunity for me to utilize my groundwork.

Recently, we were contacted by the filmmakers, asking us to process a few scenes. Initially they wanted to go with Prisma (https://youtu.be/LpsoIeOzG9I), but something didn't work out. In fact, they say they like our output more, so I'm currently busy processing 140k frames )

**Comment by ttoinou** [comment]
Looks like fun :) . 140K frames must take forever to process. Good luck !

**Comment by dovanchan** [comment]
@6o6o https://youtu.be/gq6q-xRbNxk
In you video,Can you share to us the parameters you using?or the default?

**Comment by 6o6o** [comment]
@dovanchan Mostly default, if I remember correctly. I haven't uploaded all the models, due to them being used in an app, but I likely will.

**Comment by dovanchan** [comment]
because lengstome's default is here:
![image](https://cloud.githubusercontent.com/assets/26133721/24838109/06ffcdac-1d74-11e7-9d64-23d170da2d37.png)
is different from style.py

The content weight is different,so I dont know which one is better;
I saw your result is better,so I wanna ask what's your parameters;

**Comment by 6o6o** [comment]
When I trained them 6 month ago, my defaults in `train.py` where
```
--image_size 512
--lambda_tv 10e-4
--lambda_feat 1.0
--lambda_style 10.0
--lr 1e-3
```
Those gave me best results IMO, but you shouldn't stick with them, cause it's a matter of taste. Broadly speaking, the bigger the `image_size` the better, `lambda_tv ` doesn't alter features much, only color tones. Only the ratio between `lambda_feat` and `lambda_style` matters, so you should fix the former at 1 and adjust the latter. Lower values result in less transformation.

Note: the implementation was updated with [resize-convolution](/yusuketomoto/chainer-fast-neuralstyle/tree/resize-conv) in one of the latest commits. You should use it for training new models, as it results in smoother images, without pixelated checkerboard patterns. It's located in a separate branch cause the models aren't backwards compatible. Those setting might have different effect on it.

**Comment by dovanchan** [comment]
https://www.youtube.com/watch?v=gq6q-xRbNxk&feature=youtu.be
So you say you used the default setting in this code for generating this youtube video? @6o6o

**Comment by 6o6o** [comment]
@dovanchan yes, all the models in the video were trained using above parameters
